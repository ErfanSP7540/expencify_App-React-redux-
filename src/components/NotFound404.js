import React from 'react'

const NotFound404=()=>(
    <div>
        NotFoundPage
    </div>
)

export default NotFound404;