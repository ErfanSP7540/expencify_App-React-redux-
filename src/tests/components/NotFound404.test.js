import React from 'react'
import NotFound404 from '../../components/NotFound404'

import { shallow } from 'enzyme'
import { configure } from 'enzyme'
import  Adapter  from 'enzyme-adapter-react-16'
configure({ adapter:new Adapter()});



test('404 PAGE TESTING SnapShot',()=>{
    
    const wrapper = shallow(<NotFound404 />);
    expect(wrapper).toMatchSnapshot();
})