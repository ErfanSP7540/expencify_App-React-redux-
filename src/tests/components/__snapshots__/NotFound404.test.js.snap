// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`404 PAGE TESTING SnapShot 1`] = `
<div>
  NotFoundPage
</div>
`;
